src/docs/labels/nft/index.md

```diff
@@ -2,12 +2,12 @@
 title: NFT
 author: Alex
 date: 2023-03-22
-description: NFT contract label
+description: NFT label
 ---
 
 ## Description
 
-Any smart contract which is an NFT contract should have the `nft` label assigned to it. NFT contracts follow one of two main standards: ERC721 and ERC1155. For each standard we assign a label as well.
+The NFT label is used for identifying addresses that are involved in NFT domain. This applies to NFT collections and marketplace contracts.
```

Comment:
Lets change it to (by chatgpt):

The NFT label is used to identify wallet addresses that are involved in any NFT (Non-Fungible Token) activities. By using the NFT label, you can quickly identify which wallets are involved in NFT activities and gain a better understanding of the NFT ecosystem.

NFT group includes other labels such as [NFT User](/labels/nft-user), [NFT collection](/labels/nft-collection)  and others. Visit [this page](labels/#domains) to see the full child labels list.